tpgtools/api/cloudbuildv2/samples/gle.connection.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "{{connection}}",
+  "gitlabConfig": {
+    "hostUri": "https://gle-us-central1.gcb-test.com",
+    "authorizerCredential": {
+      "userTokenSecretVersion": "projects/407304063574/secrets/gle-api-token/versions/latest"
```

Comment:
These should be secrets created within the project during sample setup rather than hard-coded secrets. What happens if these hard-coded secrets are deleted or we don't have permission on them?

Reply:
We arrived at this solution because wiring the secrets in the test code without leaking them would be complicated. We've created a shared project (gcb-terraform-creds) to hold those secrets, and the project is configured to give access to the service accounts of the projects used for running automated tests.

Note that these secrets are accessible only by the Cloud Build P4SA of the tester projects, not by the SA used for running the tests.